dsl/common/src/main/groovy/net/neoforged/gradle/dsl/common/runs/type/RunType.groovy

```diff
@@ -103,6 +104,15 @@ abstract class RunType implements ConfigurableDSLElement<RunType>, NamedDSLEleme
     @Input
     abstract Property<Boolean> getIsServer();
 
+    /**
+     * Indicates if this run is a JUnit run.
+     *
+     * @return {@code true} if this run is a JUnit run; otherwise, {@code false}.
+     */
+    @DSLProperty(propertyName = 'junit')
```

Comment:
Remove the property name. The others do not have it either.

Reply:
It is needed, otherwise the generated name is `jUnit`.

Reply:
Hmm okey
I forgot indeed that it does that
Allright thank you.